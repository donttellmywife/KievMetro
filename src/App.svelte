<script>
  export let timetable;
</script>

<h1>{ timetable.map((station => station.name)) }!</h1>

<style>
  h1 {
    color: purple;
  }
</style>
