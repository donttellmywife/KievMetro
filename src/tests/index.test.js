import { render, cleanup } from '@testing-library/svelte'
import App from '../App.svelte'
import timetable from '../data.js'

describe('App', () => {
  test('should render greeting', () => {
    const { container } = render(App, { props: { timetable } })

    expect(container.firstChild).toMatchSnapshot()
  })
})
