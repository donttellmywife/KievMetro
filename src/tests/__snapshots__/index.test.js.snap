// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render greeting 1`] = `
<div>
  <h1
    class="svelte-1ucbz36"
  >
    Сырецко-Печерская
    !
  </h1>
</div>
`;
